src/css/imports/icons.less

```diff
@@ -17,6 +17,12 @@
     .jw-icon;
 }
 
+.jw-font-load-detect {
+    position: absolute;
+    visibility: hidden;
+    display: inline-block;
+}
```

Comment:
It looks like this class is used on a test element only. Why not use inline styles?